src/helpers/augmentedReality/storageNameCreator.js

```diff
@@ -1,6 +1,13 @@
+import { documentDirectory } from 'expo-file-system';
+
 export const storageNameCreator = ({ dataItem, objectItem }) => {
-  const { id: objectId } = dataItem;
-  const { title, type, id } = objectItem;
+  const { id: objectId, title: objectTitle } = dataItem;
+  const { title, type } = objectItem;
+  const replacedObjectTitle = objectTitle.replace(/\s+/g, '');
 
-  return `${objectId}-${title}${id}.${type}`;
+  return {
+    directoryName: documentDirectory + `${replacedObjectTitle}_${objectId}/${title}.${type}`,
+    folderName: documentDirectory + `${replacedObjectTitle}_${objectId}`,
+    storageName: `${replacedObjectTitle}_${objectId}_${title}.${type}`
```

Comment:
please create variables to reduce duplications,example:

```js
1 = `${replacedObjectTitle}_${objectId}`
2 = `${title}.${type}`
...
directoryName: documentDirectory + `${1}/${2}`,
folderName: documentDirectory + 1
storageName: `${1}_${2}`
```

something like this?

Reply:
Yeah, it's better now.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/569/commits/881f40de6376a8c53355e44d9c6fe9d8dc3749f8